CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ Unless otherwise specified, any version comparison below is the comparison of se
 
 <small>[Compare with 0.1.3](https://github.com/AllayMC/Allay/compare/0.1.3...HEAD)</small>
 
+### Changed
+
+- It will skip loading failed worlds when loading worlds.
+- It will sleep a short time for gui users when abnormal exit.
```

Comment:
Change to `Main thread will sleep a short time if gui is enabled when the server exits abnormally. This gives user time to see what goes wrong.`